src/styles/styles.js

```diff
@@ -290,18 +290,16 @@ const styles = {
 
     picker: {
         inputIOS: {
-            fontFamily: fontFamily.GTA,
-            fontSize: variables.fontSizeNormal,
-            paddingLeft: 12,
-            paddingRight: 12,
-            paddingTop: 10,
-            paddingBottom: 10,
-            borderRadius: variables.componentBorderRadius,
+            flex: 1,
             borderWidth: 1,
+            borderRadius: variables.componentBorderRadiusNormal,
             borderColor: themeColors.border,
-            color: themeColors.text,
-            height: variables.inputComponentSizeNormal,
-            opacity: 1,
+            paddingTop: 25,
+            paddingHorizontal: 12,
+            paddingVertical: 8,
```

Comment:
You've got `paddingTop: 25` _and_ `paddingVertical: 8`, so one is probably overwriting the other. Definitely don't need both.

Reply:
I can change `paddingVertical` to `paddingBottom`

Reply:
Bump – change this `paddingVertical` to `paddingBottom`